sig-scheduling/2020-annual-report.md

```diff
@@ -0,0 +1,125 @@
+# 2020 SIG-Scheduling Annual Report
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md]?**
+    
+- **Is your README accurate? have a CONTRIBUTING.md file?**  
+    Yes. Developer-oriented guides are under [contributors/devel/sig-scheduling].
+
+- **All subprojects correctly mapped and listed in [sigs.yaml]?**  
+    Yes.
+
+- **What's your meeting culture? Large/small, active/quiet, learnings? 
+Meeting notes up to date? Are you keeping recordings up to date/trends 
+in community members watching recordings?**  
+    Our meetings tends to be small and quiet. The agenda usually consists of items suggested
+    by users, and debatable items that need a consensus during issues/PR reviews.
+    The meeting notes and recordings are up to date.
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any 
+springing up or being retired? Are OWNERS files up to date in these areas?**
+
+Owners from (active) subprojects introduce the latest development, and sometime demonstrate
+cool features.
+OWNER files in k/k are not that up to date. We may need a cleanup.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+Aug 20, 2020. [Slides] & [Recording].
+
+[Slides]: https://docs.google.com/presentation/d/1H27SDMqkzq8zCRveWWtK5g9hCAomKbrzTTVZ5r4h6Xo/edit
+[Recording]: https://www.youtube.com/watch?v=oDL3Kp5-9eM&feature=youtu.be
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes, except for inactive subprojects.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership, there is some churn in the number of active 
+members and so membership is hard to keep track of.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now?
+ What are you doing about it?**
+
+PRs are usually directed to the reviewer most familiar with the code base the PR is modifying.
```

Comment:
are you growing more reviewers and approvers?

Reply:
I think we have have enough approvals for the volume of the PRs we are getting, reviewers are growing.

Reply:
Yes, we're fine by now.